config/exporters/monitoring/prometheus/base/prometheus-exporter-service.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: exporter-sa
+  namespace: appstudio-exporter-service-prometheus
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: exporter-cluster-role
+rules:
+- apiGroups: ['grafana.integreatly.org']
+  resources: ['grafanas']
+  verbs: ["get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: exporter-cluster-role-binding
+  namespace: appstudio-grafana
+subjects:
+- kind: ServiceAccount
+  name: exporter-sa
+  namespace: appstudio-exporter-service-prometheus
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: exporter-cluster-role
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: exporter-service
+  namespace: appstudio-exporter-service-prometheus
+spec:
+  ports:
+  - name: https
+    port: 8090
+    targetPort: https
+  selector:
+    app: prometheus-exporter
```

Comment:
```suggestion
    app: grafana-datasource-exporter
```

Reply:
Updated to `grafana-datasource-exporter`